client/signup/steps/site-information/index.jsx

```diff
@@ -0,0 +1,218 @@
+	}
+
+	componentDidMount() {
+		SignupActions.saveSignupStep( {
+			stepName: this.props.stepName,
+		} );
+	}
+
+	handleInputChange = ( { target: { name, value } } ) => {
+		this.setState( { [ name ]: value } );
+	};
+
+	handleSubmit = event => {
+		event.preventDefault();
+
+		this.props.submitStep( this.state );
+	};
+
+	renderContent() {
+		const { translate, siteType } = this.props;
+		const siteTitleLabel = getSiteTypePropertyValue( 'slug', siteType, 'siteTitleLabel' ) || '';
+		const siteTitlePlaceholder =
+			getSiteTypePropertyValue( 'slug', siteType, 'siteTitlePlaceholder' ) || '';
+		const isBusinessSiteSelected =
+			siteType === getSiteTypePropertyValue( 'slug', 'business', 'slug' );
+
+		return (
+			<div className="site-information__wrapper">
+				<div className="site-information__form-wrapper ">
+					<form>
+						<Card>
+							<FormFieldset>
+								<FormLabel htmlFor="name">
+									{ siteTitleLabel }
+									<InfoPopover className="site-information__info-popover" position="top">
+										{ translate(
+											"We'll use this as your site title. Don't worry, you can change this later."
+										) }
+									</InfoPopover>
+								</FormLabel>
+								<FormTextInput
+									id="name"
+									name="name"
+									placeholder={ siteTitlePlaceholder }
+									onChange={ this.handleInputChange }
+									value={ this.state.name }
+								/>
+							</FormFieldset>
+
+							{ isBusinessSiteSelected && (
+								<>
+									<FormFieldset>
+										<FormLabel htmlFor="address">
+											{ translate( 'Address' ) }
+											<InfoPopover className="site-information__info-popover" position="top">
+												{ translate( 'Where can people find your business?' ) }
+											</InfoPopover>
+										</FormLabel>
+										<FormTextarea
+											id="address"
+											name="address"
+											placeholder={ 'E.g. 60 29th Street #343\nSan Francisco, CA\n94110' }
+											onChange={ this.handleInputChange }
+											value={ this.state.address }
+										/>
+									</FormFieldset>
+									<FormFieldset>
+										<FormLabel htmlFor="email">
+											{ translate( 'Contact Email' ) }
+											<InfoPopover className="site-information__info-popover" position="top">
+												{ translate( 'Does your business have an email address?' ) }
+											</InfoPopover>
+										</FormLabel>
+										<FormTextInput
+											id="email"
+											name="email"
+											type="email"
+											placeholder={ 'E.g. email@domain.com' }
+											onChange={ this.handleInputChange }
+											value={ this.state.email }
+										/>
+									</FormFieldset>
+									<FormFieldset>
+										<FormLabel htmlFor="phone">
+											{ translate( 'Phone number' ) }
+											<InfoPopover className="site-information__info-popover" position="top">
+												{ translate( 'How can people contact you?' ) }
+											</InfoPopover>
+										</FormLabel>
+										<FormTelInput
```

Comment:
FYI, we also have `FormPhoneMediaInput`: https://github.com/Automattic/wp-calypso/blob/7a48ac32ab305c8417c1f8a09b55b6a755607651/client/components/forms/form-phone-media-input/index.jsx

<img width="1028" alt="screenshot 2018-11-26 at 09 39 09" src="https://user-images.githubusercontent.com/3392497/49005655-27fed780-f15f-11e8-9359-c06e54f2bc67.png">

See it live here https://wpcalypso.wordpress.com/devdocs/design/form-fields - it's also used in the domain contact form during checkout. It tries to accept any way the user wants to format the phone and formats/sanitizes it consistently.

Reply:
@klimeryk thanks for the pointer! I think since this will be used as-supplied on the front-end of the site, we don't want to force the country code part of the format, but this is good to know for possible future iterations.

Reply:
Ah, good point - in this context, users might want to force a certain format that they're most familiar with (or think is "proper"). On the other hand, the autoformatting that this field offers should ensure the phone is in a format that Google would be able to easily scrape and understand. But, yeah, that might be best left for future iterations if you notice that's a problem 👍